avaframe/log2Report/generateReport.py

```diff
@@ -12,6 +12,34 @@
 log = logging.getLogger(__name__)
 
 
+
+def addLineBlock(titleString, reportDKey, pfile, italicFont=False, flagBreak=False):
+    """ add lineblock to report
+
+        Parameters
+        -----------
+        titleString: str
+            string that shall be added
+        reportDKey: dict
+            dictionary with info for string
+        pfile: file
+        italicFont: bool
+            if True write value in italic
+        flagBreak: bool
+            True break if if conidtion is met
```

Comment:
? Typos?
And I don't get the functionality of this flag

Reply:
I changed this to: 
flagBreak: bool
            if first item in reportDKey is added to pfile - break